pdf/src/campPrint/cover/Cover.vue

```diff
@@ -1,15 +1,16 @@
 <template>
   <Page :id="id" size="A4" :bookmark="$tc('print.cover.title')" class="page">
-    <Text class="cover-camp-name cover-center">{{ config.camp.name }}</Text>
-    <Text class="cover-camp-title cover-center">
-      {{ config.camp.title }}
-    </Text>
-    <Text class="cover-camp-motto cover-center">
-      {{ config.camp.motto }}
-    </Text>
-    <Text v-if="config.camp.organizer" class="cover-camp-organizer cover-center">
-      {{ config.camp.organizer }}
-    </Text>
+    <View class="cover-camp-wrapper">
+      <Text v-if="config.camp.organizer" class="cover-camp-organizer cover-center">
+        {{ config.camp.organizer }}
+      </Text>
+      <Text class="cover-camp-title cover-center">
+        {{ config.camp.title ?? config.camp.shortname }}
```

Comment:
In the issue we said that the shortname should always be displayed. If we have place, we also show the title